plugin.xml

```diff
@@ -8,6 +8,7 @@
     <engine name="cordova" version=">=4.0.0"/>
   </engines>
   <dependency id="cordova-plugin-file" version=">=2.0.0"/>
+  <preference name="AndroidBlacklistTlsProtocols" default="SSLv3,TLSv1"/>
```

Comment:
Could we call this `AndroidBlacklistSecureSocketProtocols`, so it's more general (like in the [Android docs](https://developer.android.com/reference/javax/net/ssl/SSLSocket#default-configuration-for-different-android-versions))? It doesn't only apply to TLS and maybe in the future there will be another protocol besides TLS and SSL.